.github/workflows/fix-style.yml

```diff
@@ -1,17 +1,22 @@
 name: "Fix Style"
 on:
-  issue_comment:
-    types: [created]
+  #issue_comment:
+  #  types: [created]
+  push:
 
 jobs:
   fix-style:
-    if: github.event.issue.pull_request && startsWith(github.event.comment.body, 'fix style')
+    #if: github.event.issue.pull_request && startsWith(github.event.comment.body, 'fix style')
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
-      - uses: pre-commit/action@v3.0.0
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
```

Comment:
TODO: flake8 breaks if we use Python 3.10 because of changes in the parser. I think we may need to update flake8 and add the flake8-force to fix this? https://github.com/kmaehashi/flake8-force